packages/design-system/src/components/Popover/Popover.stories.tsx

```diff
@@ -3,11 +3,26 @@ import React from 'react';
 import Popover from './Popover';
 import { ButtonPrimary } from '../Button';
 import { action } from '@storybook/addon-actions';
+import { PopoverStateReturn } from 'reakit/ts';
+import { StackVertical } from '../Stack';
 
 export default {
 	component: Popover,
 };
 
+function EasyPopover() {
+	return <StackVertical gap="S">Hello hello</StackVertical>;
+}
+
+function ControlledPopover({ popover }: { popover?: PopoverStateReturn }) {
```

Comment:
```suggestion
function ChildrenAsFunctionPopoverContent({ popover }: { popover?: PopoverStateReturn }) {
```

Reply:
The Popover is still uncontrolled, when we think about it. Its state is still entirely self-contained.